packages/trader/src/Modules/SmartChart/Components/buy-toast-notification.jsx

```diff
@@ -0,0 +1,57 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { MobileWrapper, Toast, Text } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import { connect } from 'Stores/connect';
+import { findContractCategory } from '../../Trading/Helpers/contract-type';
+
+const BuyToastNotification = ({ portal_id = 'popup_root', action_toastbox, actionChangeToastbox }) => {
+    const { buy_price, currency, contract_type, list } = action_toastbox;
+    const active_trade_type = { value: contract_type };
+
+    const trade_type_name = findContractCategory(list, active_trade_type)?.contract_types?.find(
+        item => item.value === contract_type
+    ).text;
+
+    const message = (
+        <Text as='p' size='xxs' className='dc-toast__notification'>
+            <Localize
+                i18n_default_text='The purchase of <0>{{trade_type_name}} contract</0> has been completed for the amount of <0> {{buy_price}} {{currency}}</0>'
+                components={[<strong key={0} />]}
+                values={{ trade_type_name, buy_price, currency }}
+                shouldUnescape={true}
```

Comment:
```suggestion
                shouldUnescape
```

Reply:
Can you explain on why do you need `shouldUnescape`

Reply:
Because when i am passsing string to Localize `/` is converted into Html Unicode that's why i have to use Unescape to get the content instead of Unicode (Rise/Fall)